.github/workflows/release.yml

```
@@ -0,0 +1,24 @@
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+      - name: Install semantic-release
+        run: |
+          npm init -y >/dev/null 2>&1 || true
+          npm install -D semantic-release @semantic-release/changelog @semantic-release/git @semantic-release/github
+      - name: Run semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: npx semantic-release
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)